package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "com.disguise.renderstream",
+  "displayName": "Disguise RenderStream",
+  "version": "1.0.0",
+  "unity": "2023.2",
```

Comment:
That's the minimal version?  Is 2023.2 released yet?

Reply:
Changed to 2022.2.6f1 which I've used a lot during dev with no issues. If we go too far back (ex 2022.2.0b16) some of the API is missing so I'll keep it at 6f1 for now.